crates/chia-puzzles/src/puzzles/cat.rs

```diff
@@ -250,77 +223,4 @@ mod tests {
         assert_puzzle_hash!(EVERYTHING_WITH_SIGNATURE_TAIL_PUZZLE => EVERYTHING_WITH_SIGNATURE_TAIL_PUZZLE_HASH);
         assert_puzzle_hash!(GENESIS_BY_COIN_ID_TAIL_PUZZLE => GENESIS_BY_COIN_ID_TAIL_PUZZLE_HASH);
     }
-
-    #[test]
-    fn curry_cat_tree_hash() {
-        let inner_args = StandardArgs {
-            synthetic_key: PublicKey::default(),
-        };
-        let asset_id = Bytes32::new([120; 32]);
-
-        let mut a = Allocator::new();
-        let mod_ptr = node_from_bytes(&mut a, &CAT_PUZZLE).unwrap();
-        let inner_mod_ptr = node_from_bytes(&mut a, &STANDARD_PUZZLE).unwrap();
-        let curried_ptr = CurriedProgram {
-            program: mod_ptr,
-            args: CatArgs {
-                mod_hash: CAT_PUZZLE_HASH.into(),
-                inner_puzzle: CurriedProgram {
-                    program: inner_mod_ptr,
-                    args: &inner_args,
-                },
-                tail_program_hash: asset_id,
-            },
-        }
-        .to_node_ptr(&mut a)
-        .unwrap();
-
-        let expected_tree_hash = hex::encode(tree_hash(&a, curried_ptr));
-        let actual_tree_hash = hex::encode(cat_puzzle_hash(
-            asset_id,
-            standard_puzzle_hash(&inner_args.synthetic_key),
-        ));
-
-        assert_eq!(expected_tree_hash, actual_tree_hash);
-    }
-
-    #[test]
-    fn curry_everything_with_signature() {
```

Comment:
I would think we would still want these tests, just using the new mechanism to compute the hashes

Reply:
I can add them back if you'd like, but it might make sense to have more comprehensive tests for all of the puzzles if so, to be consistent. I got rid of these since there's nothing special with functions like `cat_puzzle_hash` now.